rllib/algorithms/algorithm.py

```diff
@@ -1658,7 +1637,8 @@ def add_policy(
             ]
         ] = None,
         evaluation_workers: bool = True,
-        workers: Optional[List[Union[RolloutWorker, ActorHandle]]] = None,
+        # Deprecated.
+        workers: Optional[List[Union[RolloutWorker, ActorHandle]]] = DEPRECATED_VALUE,
```

Comment:
Why are we deprecating this?
What's the replacement for someone wanting to add certain policies only to some of the remote workers?

Reply:
Ok, after internal discussion: Fine with me!
* We did not support keeping track of which workers had which policies for a) crashes and b) checkpoints before anyways.
* We should address the concern about memory consumption of workers (due to too many policies on each worker) via the in-flight PR for the enhanced LRU cache used by PolicyMap, anyways.

Reply:
👍 😄